### PR TITLE
http-client: add body form urlencoded option

### PR DESCRIPTION
**Pull request by RonaldEAM**

### CODE REVIEW

File: packages/integration-sdk-http-client/src/client.ts [hunk 1]
```diff
@@ -243,6 +243,17 @@ export abstract class BaseAPIClient {
         });
       } else if (bodyType === 'text' && typeof body === 'string') {
         fmtBody = body;
+      } else if (bodyType === 'urlencoded') {
+        fmtBody = new URLSearchParams();
+        Object.entries(body).forEach(([key, value]) => {
+          if (typeof value !== 'string') {
+            throw new IntegrationError({
```
Comment (VDubber) [inline packages/integration-sdk-http-client/src/client.ts:250]: should we provide a softer interface that allows other types but we handle the conversion? Thoughts?

Comment (RonaldEAM) [reply packages/integration-sdk-http-client/src/client.ts:250]: there's not too many options left, only binary, and I don't think we will ever upload files ever,  it's ok to keep it like this for now

File: packages/integration-sdk-http-client/src/client.ts
[the remark this answers is not in the document]
Comment (RonaldEAM) [reply packages/integration-sdk-http-client/src/client.ts:1]: added!

File: packages/integration-sdk-http-client/src/__tests__/index.test.ts [hunk 2]
```diff
@@ -179,6 +181,155 @@ describe('APIClient', () => {
         body: `{"test":"test"}`,
       });
     });
+
+    describe('fmtBody', () => {
```
Comment (VDubber) [inline packages/integration-sdk-http-client/src/__tests__/index.test.ts:185]: ❤️

**Review by VDubber — APPROVED** [review]

**Pull request merged**